systemtests/src/main/java/io/enmasse/systemtest/utils/IoTUtils.java

```diff
@@ -196,6 +189,32 @@ private static void waitForIoTProjectDeleted(Kubernetes kubernetes, IoTProject p
             return updated == null;
         }, Duration.ofMinutes(5), Duration.ofSeconds(10), () -> "IoT project failed to delete in time");
         log.info("IoTProject {}/{} deleted", project.getMetadata().getNamespace(), project.getMetadata().getName());
+
+        // get the expected and actual address spaces
+
+        var projectClient = kubernetes.getIoTProjectClient(project.getMetadata().getNamespace());
```

Comment:
You already have the project client above.

Reply:
Indeed. Fixed that.